docs/index.rst

```diff
@@ -530,82 +544,4 @@ Relevant configuration values
 SpreadSASLMemcachedCache
 ````````````````````````
 
-Set ``CACHE_TYPE`` to ``spreadsaslmemcached`` to use this type.
```

Comment:
Please restore this part of the documentation again

Reply:
Done :-)